frontend/src/components/projects/NewProject.tsx

```diff
@@ -3,13 +3,19 @@ import LoadingSpinner from "../Spinners/LoadingSpinner";
 
 import NewProjectForm from "./NewProjectForm";
 
+// interface Props {
```

Comment:
nit: omitting commented-out code makes the PR easier to review

Reply:
Removed comments